main.tf

```diff
@@ -76,7 +88,7 @@ resource "azurerm_template_deployment" "app_service_site" {
   count               = "${var.enable_ase}"
   template_body       = "${data.template_file.sitetemplate.rendered}"
   name                = "${var.product}-${var.env}${var.deployment_target}-webapp"
-  resource_group_name = "${local.resource_group_name}"
+  resource_group_name = "${azurerm_resource_group.rg.name}"
```

Comment:
please revert this change, we're in the progress of migrating applications to always passing in a resource group to the module, as we're decommissioning this module as part of moving to Kubernetes

REQ

Reply:
https://github.com/hmcts/cnp-module-webapp/pull/156#discussion_r322741012